rollout.go

```diff
@@ -64,29 +62,32 @@ func (r *client) Stop() {
 
 func (r *client) poll(path string) {
 	defer func() { r.done <- true }()
-	defer log.Println("Rollout poller shutdown")
+	defer log.Info("Rollout poller shutdown")
 	for {
 		data, _, watch, err := r.zk.GetW(path)
 		if err != nil {
-			log.Println("Rollout: Failed to set watch", err)
+			log.Error("Rollout: Failed to set watch", err)
```

Comment:
I think these need a trailing space in the first argument to avoid bleeding `watch` into the first word of the error string?

`log.Println` calls `fmt.Sprintln` to format its arguments (https://golang.org/src/log/log.go?s=9152:9182#L290) while `logrus.Enyry.Error` uses `fmt.Sprint` to format is arguments (https://github.com/sirupsen/logrus/blob/ea8897e79973357ba785ac2533559a6297e83c44/entry.go#L167). `Sprintln` always separates arguments with spaces while `Sprint` only does it for non-string pairs of arguments.